docs/platform/compliance/compliance-specification.md

```diff
@@ -16,7 +16,7 @@ The Tizen Compliance Specification is the definitive set of requirements for sof
 
 - The following specification is the beta version for Tizen TV profile 5.5 devices:
 
-  [Tizen 5.5 Compliance Specification for TV Profile](media/tizen-5.5-compliance-specification-for-tv-profile-v0.95beta.pdf) ![PDF icon](media/application-pdf.png)
+  [Tizen 5.5 Compliance Specification for TV Profile](media/tizen-5.5-compliance-specification-for-tv-profile-v0.96beta.pdf) ![PDF icon](media/application-pdf.png)
```

Comment:
I just opened tizen-5.5-compliance-specification-for-tv-profile-v0.96beta.pdf file and could see below updated in the file.

I'm not sure whether this update is intended, so hope to get your confirmation for below.
![image](https://user-images.githubusercontent.com/9958374/83587552-564a4800-a58a-11ea-8d90-d5e4da6a4949.png)

...
![image](https://user-images.githubusercontent.com/9958374/83587655-8396f600-a58a-11ea-970a-42ff448dffca.png)

Reply:
I've also found and removed the unintended spaces in the TOC, which were unexpectedly inserted with "field update". It looks trivial but I'll make the fixes reflected again.